actionpack/lib/action_dispatch/middleware/templates/rescues/_message_with_did_you_mean.html.erb

```diff
@@ -0,0 +1,10 @@
+<% message, did_you_mean_suggestions = @exception.message.split(/Did you mean\?/, 2) %>
```

Comment:
Any way to pass it separated (not joined before)? If not, at least test would be nice to cover this would not break later.

Reply:
DidYouMean supports calling the `original_message` and the `corrections` on the exception. I've changed  the code.

Reply:
Maybe `message_and_corrections` is a better name for the template.